app/models/audits/report_row.rb

```diff
@@ -29,7 +29,6 @@ def precompute
         other_organisations,
         content_type,
         last_major_update,
-        whitehall_url,
```

Comment:
`whitehall_url` is currently being displayed on the report (CSV). Could you please double check with Ben that this is no longer needed? It seems important to be able to do the changes quickly, but I might be missing something.